### ISSUE TITLE
Installation Error: `make` failed with exit code: 2

### ISSUE THREAD

**Issue by Bowfish** [ethereum/web3.js#2992]
If I run `npm install web3 --save` I get the following error messages:

```
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/xyz/.nvm/versions/node/v12.7.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:196:23)
gyp ERR! stack     at ChildProcess.emit (events.js:203:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:272:12)
gyp ERR! System Linux 5.2.3-1-default
gyp ERR! command "/home/xyz/.nvm/versions/node/v12.7.0/bin/node" "/home/xyz/.nvm/versions/node/v12.7.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/xyz/src/truffle-new/box-drizzle/app/node_modules/scrypt
gyp ERR! node -v v12.7.0
gyp ERR! node-gyp -v v5.0.3
gyp ERR! not ok 
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: scrypt@6.0.3 (node_modules/scrypt):
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: scrypt@6.0.3 install: `node-gyp rebuild`
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: Exit status 1
```
I tried about everything I found about this issue, none of them worked. I even reinstalled nvm and node. 

Operating system: linux
node version: 12.7.0
npm version: 6.10.2

**Comment by nivida** [comment]
This got fixed with https://github.com/ethereum/web3.js/pull/2938 and will be released with [1.2.1](https://github.com/ethereum/web3.js/blob/1.x/CHANGELOG.md).

**Comment by lucianosilvi** [comment]
Hi, when do you estimate the version 1.2.1 will be released?
Thanks
